src/AFQMC/Hamiltonians/HSPotential_Helpers.h

```diff
@@ -23,15 +23,17 @@ namespace afqmc
 {
 namespace HamHelper
 {
+template<class T>
```

Comment:
I think making `template<bool IS_COMPLEX>` reduce the complexity of explicit template instantiation cases.